app/src/main/java/org/mozilla/fenix/ext/BrowserState.kt

```diff
@@ -24,16 +24,9 @@ import kotlin.math.max
 fun BrowserState.asRecentTabs(): List<RecentTab> {
     return mutableListOf<RecentTab>().apply {
         val lastOpenedNormalTab = lastOpenedNormalTab
-        val inProgressMediaTab = inProgressMediaTab
```

Comment:
We could also wrap this behind a feature flag?

Reply:
Yeah, I wasn't sure to what extent we should be "removing it". So I opted for this solution so we can land it before the beta cut.

Reply:
Another reason I forgot about, is that we get intermittent test failures when we have feature flags here. See https://github.com/mozilla-mobile/fenix/pull/21557.

Reply:
Let's land as is. Thought about this last week and it's a trade-off. We don't want feature flags to be too fine-grained, so arguably this is a change in behaviour of an existing features as opposed to a feature itself. We can revert this to enable again when we want it (have time to add media controls).